src/frontend/pychiquito.rs

```diff
@@ -68,7 +68,7 @@ impl<'de> Visitor<'de> for CircuitVisitor {
                     if exposed.is_some() {
                         return Err(de::Error::duplicate_field("exposed"));
                     }
-                    exposed = Some(map.next_value::<Vec<ForwardSignal>>()?);
+                    exposed = Some(map.next_value::<Vec<(Queriable<Fr>, ExposeOffset)>>()?);
```

Comment:
@qwang98 this is where the breaking change lies. Queriables do not implement any serde traits.

Reply:
I merged over this branch and I'm working on this :)